src/Http/WebUtilities/src/ReasonPhrases.cs

```diff
@@ -88,6 +182,14 @@ public static class ReasonPhrases
     /// <returns>The reason phrase, or <see cref="string.Empty"/> if the status code is unknown.</returns>
     public static string GetReasonPhrase(int statusCode)
```

Comment:
We should replace https://github.com/dotnet/aspnetcore/blob/main/src/Servers/HttpSys/src/RequestProcessing/HttpReasonPhrase.cs

Reply:
I could do that, but keep in mind, that the following StatusCodes will then start returning a result:
| StatusCode | ReasonPhrase                    |
| ---------- | ------------------------------- |
| 208        | Already Supported               |
| 226        | IM Used                         |
| 308        | Permanent Redirect              |
| 418        | I'm a teapot                    |
| 419        | Authentication Timeout          |
| 421        | Misdirected Request             |
| 428        | Precondition Required           |
| 429        | Too Many Requests               |
| 431        | Request Header Fields Too Large |
| 451        | Unavailable For Legal reasons   |
| 499        | Client Closed Request           |
| 506        | Variant Also Negotiates         |
| 508        | Loop Detected                   |
| 510        | Not Extended                    |
| 511        | Network Authentication Required |